### PR TITLE
Add dartdoc to Tooltip

### PR DESCRIPTION
**Pull request by abarth**
Also, remove several unused configuration options and fix an animation
leak.

### CODE REVIEW

**Comment by abarth** [comment]
@Hixie

File: packages/flutter/lib/src/material/tooltip.dart [hunk 1]
```diff
@@ -8,76 +8,75 @@ import 'dart:math' as math;
...
-
+import 'typography.dart';
+
+const double _kScreenEdgeMargin = 10.0;
+const Duration _kFadeDuration = const Duration(milliseconds: 200);
+const Duration _kShowDuration = const Duration(seconds: 2);
+
+/// A material design tooltip.
+///
+/// Tooltips provide text lables that helps explain the function of a button or
```
Comment (Hixie) [inline packages/flutter/lib/src/material/tooltip.dart]: labels

Comment (Hixie) [inline packages/flutter/lib/src/material/tooltip.dart]: help

**Comment by Hixie** [comment]
LGTM

**Pull request merged**